### PR TITLE
B 1783 features ecosystem sliders

### PR DESCRIPTION
**Pull request by alexbeno**

### CODE REVIEW

File: app/ecosystems/features/featured-ecosystems/components/slide/slide.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+export function Slide({ imageUrl, slug, color, title, description }: TSlide.Props) {
+  const slideColor = getBannerColor({ color });
+  return (
+    <a
+      href={NEXT_ROUTER.ecosystems.details.root(slug)}
+      className="outline-st flex h-full w-full flex-col items-start justify-end rounded-[32px] bg-card-background-base bg-cover bg-center px-16 py-12 outline outline-[6px] outline-card-border-medium"
+      style={{
+        backgroundImage: `url(${imageUrl})`,
+      }}
```
Comment (haydencleary) [inline app/ecosystems/features/featured-ecosystems/components/slide/slide.tsx:19]: Could use an `img` tag and position it in the background, that way we could lazy load the asset

File: app/ecosystems/features/featured-ecosystems/components/slider/slider.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      return (
+        <button
+          key={slideId}
+          onClick={() => {
+            instanceRef.current?.moveToIdx(slideId);
+          }}
+          className={cn("h-2 w-2 rounded-full bg-spaceBlue-400", {
+            "!bg-greyscale-50": currentSlide === slideId,
+          })}
+        ></button>
```
Comment (haydencleary) [inline app/ecosystems/features/featured-ecosystems/components/slider/slider.tsx:42]:
```suggestion
        />
```

File: app/ecosystems/features/featured-ecosystems/components/slider/slider.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    <>
+      <div ref={sliderRef} className={cn("keen-slider transition-all", { "pointer-events-none opacity-0": !loaded })}>
+        {children.map((c, key) => (
+          <div key={key} className="keen-slider__slide o h-[360px] bg-transparent p-1.5">
+            {c}
+          </div>
+        ))}
+      </div>
+      <div className="flex w-full flex-row items-center justify-center gap-2 p-4">
+        {navigationsElements.map(el => el)}
```
Comment (haydencleary) [inline app/ecosystems/features/featured-ecosystems/components/slider/slider.tsx:57]: I don't think the map is necessary here ?

File: app/ecosystems/features/featured-ecosystems/components/slider/slider.tsx [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+        ></button>
+      );
+    });
+  }, [navigationsItems]);
+
+  return (
+    <>
+      <div ref={sliderRef} className={cn("keen-slider transition-all", { "pointer-events-none opacity-0": !loaded })}>
+        {children.map((c, key) => (
+          <div key={key} className="keen-slider__slide o h-[360px] bg-transparent p-1.5">
```
Comment (haydencleary) [inline app/ecosystems/features/featured-ecosystems/components/slider/slider.tsx:51]:
```suggestion
          <div key={key} className="keen-slider__slide h-[360px] bg-transparent p-1.5">
```

File: app/ecosystems/features/featured-ecosystems/components/slide/slide.tsx (hunk 1 above)
Comment (alexbeno) [reply app/ecosystems/features/featured-ecosystems/components/slide/slide.tsx:19]: can do that

File: app/ecosystems/features/featured-ecosystems/components/slider/slider.tsx (hunk 3 above)
Comment (alexbeno) [reply app/ecosystems/features/featured-ecosystems/components/slider/slider.tsx:57]: linter error :/

**Review by haydencleary — APPROVED** [review]

**Pull request merged**